examples/api-gateway-rest-api-openapi/outputs.tf

```diff
@@ -0,0 +1,15 @@
+#
+# Outputs
+#
+
+output "domain_url" {
```

Comment:
Nit: These are `curl` commands to access the URL, not just the URL. For accuracy of naming, it might be good to rename the outputs

Reply:
Good point! I'll adjust these.